IT-Wizards-API/src/main/java/org/LaunchCode/IT_Wizards_API/models/ItemCategory.java

```diff
@@ -32,7 +33,8 @@ public void setName(String name) {
         this.name = name;
     }
 
-    //Methods
+    public List<Item> getItems() {return items;}
```

Comment:
For some reason this line of code prevents the ItemCategoriesPage from displaying. After I comment it out and rerun the API, the page displays normally again. I'm not sure why this is.